QA.md

```diff
@@ -124,277 +124,3 @@
     1. We should never have any problems with pt-br translation. We should have all texts translated.
     1. Check if all untranslated texts are known.
 
```

Comment:
I like to split the QA in different files, I would just add a link do the other files, so we never forget to do it when following the QA.

```
# Push Notification

Follow the QA steps for push notifications here.

# Custom Network

Follow the QA steps for custom network here.
```

Something similar do that. What do you think?

Reply:
I like call the QA "Custom Network", so I renamed to file: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/372/commits/3e7ae3570c5224625b8f7d923e7eac6bedf51d3d

I added the links at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/372/commits/14fab0befecfe3c3a002019071de7187720f78ba